src/helpers/file.helpers.ts

```diff
@@ -19,6 +19,27 @@ const isConfigured = (config: Config) => {
     isAuthConfigured = isAuthUsernameConfigured && isAuthPasswordConfigured;
   }
   return isProjectKeyConfigured && isSonarURLConfigured && isAuthConfigured;
+}
+
+export function isPullRequestOrBranchConfigured(config: Config) {
+  let returnVal = {};
+
+  let isBranchConfigured = has(config, 'branch') && !config.branch.includes('optional-branch-name');
+  let isPullRequestConfigured = has(config, 'pullRequest') && !config.pullRequest.includes('optional-pull-request-id');
+
+  if ( isBranchConfigured || isPullRequestConfigured) {
+    if ( isBranchConfigured ) {
+      returnVal = { branch: config.branch };
+    }
+
+    // Give pullRequest highest priority
```

Comment:
if you want to give highest priority you can move before the `if ( isBranchConfigured ) {` and return directly

Reply:
Hi @pedroresende ,
I refactored changes. Thanks for the tip.